buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -187,13 +186,13 @@ private static void createUseClusterTaskExtension(Project project) {
                 .set(
                     "useCluster",
                     new Closure<Void>(project, task) {
```

Comment:
FWIW, here's an example where static Groovy sure would be nicer than plain Java 😉